languages/python/reference/exercise-concepts/markdown.md

```diff
@@ -0,0 +1,48 @@
+# Concepts for markdown
+
+[Example Implementation](https://github.com/exercism/python/blob/master/exercises/markdown/example.py)
+
+## General  
+
+
+- functions: This exercise requires a re-write of pre-esiting code that uses functions to parse passed-in text in _markdown_.  Tests expect a function named `parse` that can be called to transform the _markdown_ formatted text and return HTML formatted text.
+
+- function arguments:  The example solution uses multiple functions that take various function arguments to operate on different pieces of the passed in markdown string.
+
+- `re` module:  Both the original code to be refactored for this exercise and the example solution import and use the `re` module for Regular Expressions in python.
+
+- `import` key word/ importing:  Both the original code to be refactored for the exercise and the example solution use the `import` keyword to import the `re` module in support of Regular Expressions in python.
+
+- `str.format()` :  The example solution uses `str.split()` to break the passed in markdown string into a list of lines broken up by the `\n` character.
```

Comment:
Should the concept not be `str.split` here, rather than `str.format`? This is an example of where the example.py isn't very idiomatic, that really should be `str.splitlines` in the real world.

Reply:
Fixed.

> - `str.splitlines()`/`str.split()` :  The example solution uses `str.split()` to break the passed in markdown string into a list of lines broken up by the `\n` character.  The alternate Python example solution uses `str.splitlines() for the same effect across all line end characters.